sdk/src/main/java/com/meniga/sdk/models/offers/reimbursementaccounts/MenigaReimbursementAccount.java

```diff
@@ -79,6 +79,10 @@ public void setAccountType(String accountType) {
 		this.accountType = accountType;
 	}
 
+	public String getAccountInfo() {
```

Comment:
According to swagger spec this can't be null, returns `@NonNull` can be added then.

Reply:
Addedd